API-strategie-modulen/API-strategie-mod-geo/request-response.md

```diff
@@ -110,7 +113,7 @@ In a REST API that uses JSON as the data format, the geometry is returned as a G
     }
   }</pre>
   <h4 class="rulelab">How to test</h4>
-  <p>Validate that the returned document represents coordinates using: </p>
+  <p>Issue an HTTP GET request to the API. Validate that the returned document represents coordinates using: </p>
```

Comment:
Zoals gespecificeerd in http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/geometryGeoJSON.yaml

Reply:
done